corehq/apps/es/const.py

```diff
@@ -30,44 +30,45 @@
 }
 
 HQ_APPS_INDEX_CANONICAL_NAME = "apps"
-HQ_APPS_INDEX_NAME = "apps-20230524"
-HQ_APPS_SECONDARY_INDEX_NAME = "apps-2024-05-09"
+HQ_APPS_INDEX_NAME = "apps-2024-05-09"
+HQ_APPS_SECONDARY_INDEX_NAME = None
 
 HQ_CASE_SEARCH_INDEX_CANONICAL_NAME = "case_search"
-HQ_CASE_SEARCH_INDEX_NAME = "case-search-20230524"
-HQ_CASE_SEARCH_SECONDARY_INDEX_NAME = "case-search-2024-05-09"
+HQ_CASE_SEARCH_INDEX_NAME = "case-search-2024-05-09"
+HQ_CASE_SEARCH_SECONDARY_INDEX_NAME = None
 
 # See case_search_bha.py docstring for context
 HQ_CASE_SEARCH_BHA_INDEX_CANONICAL_NAME = "case_search_bha"
 HQ_CASE_SEARCH_BHA_INDEX_NAME = "case-search-bha-2024-05-10"
 HQ_CASE_SEARCH_BHA_SECONDARY_INDEX_NAME = None
 
 HQ_CASES_INDEX_CANONICAL_NAME = "cases"
-HQ_CASES_INDEX_NAME = "cases-20230524"
-HQ_CASES_SECONDARY_INDEX_NAME = "cases-2024-05-09"
+HQ_CASES_INDEX_NAME = "cases-2024-05-09"
+HQ_CASES_SECONDARY_INDEX_NAME = None
 
 HQ_DOMAINS_INDEX_CANONICAL_NAME = "domains"
-HQ_DOMAINS_INDEX_NAME = "domains-20230524"
-HQ_DOMAINS_SECONDARY_INDEX_NAME = "domains-2024-05-09"
+HQ_DOMAINS_INDEX_NAME = "domains-2024-05-09"
+HQ_DOMAINS_SECONDARY_INDEX_NAME = None
 
 HQ_FORMS_INDEX_CANONICAL_NAME = "forms"
-HQ_FORMS_INDEX_NAME = "forms-20230524"
-HQ_FORMS_SECONDARY_INDEX_NAME = "forms-2024-05-09"
+HQ_FORMS_INDEX_NAME = "forms-2024-05-09"
+HQ_FORMS_SECONDARY_INDEX_NAME = None
 
 HQ_GROUPS_INDEX_CANONICAL_NAME = "groups"
-HQ_GROUPS_INDEX_NAME = "groups-20230524"
-HQ_GROUPS_SECONDARY_INDEX_NAME = "groups-2024-05-09"
+HQ_GROUPS_INDEX_NAME = "groups-2024-05-09"
+HQ_GROUPS_SECONDARY_INDEX_NAME = None
 
 HQ_SMS_INDEX_CANONICAL_NAME = "sms"
-HQ_SMS_INDEX_NAME = "sms-20230524"
-HQ_SMS_SECONDARY_INDEX_NAME = "sms-2024-05-09"
+HQ_SMS_INDEX_NAME = "sms-2024-05-09"
+HQ_SMS_SECONDARY_INDEX_NAME = None
 
 HQ_USERS_INDEX_CANONICAL_NAME = "users"
-HQ_USERS_INDEX_NAME = "users-20230524"
-HQ_USERS_SECONDARY_INDEX_NAME = "users-2024-05-09"
+HQ_USERS_INDEX_NAME = "users-2024-05-09"
+HQ_USERS_SECONDARY_INDEX_NAME = None
 
 ES_REINDEX_LOG = [
     '6',
+    '7'
```

Comment:
Can you elaborate on adding '7' to the reindex log? It feels like this shouldn't be a necessary step in the process of finalizing an upgrade to 6 right?

Reply:
so this setting ensures that the multiplex settings are rendered useless. If this is not updated, then there are chances that someone might have not updated their reindex settings like their index swapped is set to True. And if we try to use that after we have set secondary indexes to null this would cause an issue. So thats why it is required.